Documentation/gettingstarted.rst

```diff
@@ -701,19 +701,7 @@ of the same TCP/UDP connection.
 
 We can achieve that with the following Cilium policy:
 
-::
-
-    [{
-        "endpointSelector": {"matchLabels":{"id":"app1"}},
-        "ingress": [{
-            "fromEndpoints": [
-                {"matchLabels":{"id":"app2"}}
-            ],
-            "toPorts": [{
-                    "ports": [{"port": "80", "protocol": "TCP"}]
-            }]
-        }]
-    }]
+.. literalinclude:: ../examples/policies/cilium_dkr_demo_l3-l4-policy-170817.json
```

Comment:
Can you move the files into `examples/policies/getting-started/`?

Any reason for the date in the filename?

Reply:
No, not really, was using unique nearby string I saw. Can change it something else, just let me know what? Moving it into the new directory `examples/policies/getting-started/`.